automatic/windows10-media-creation-tool/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,23 @@
+﻿$ErrorActionPreference = 'Stop'
+$exeName = "MediaCreationTool.exe"
+
+$toolsDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+  PackageName  = $env:chocolateyinstall
+  FileFullPath = "$toolsDir\$exeName"
+  Url          = ''
+  Checksum     = ''
+  ChecksumType = ''
+}
+Get-ChocolateyWebFile @packageArgs
+
+$AppPathKey = "Registry::HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\$($exeName)"
```

Comment:
This entire section should be replaced with `Register-Application` from core-extension.